packages/ui-library/src/components/modal/index.tsx

```diff
@@ -38,28 +38,33 @@ const StyledDialogTitle = styled.div`
 `;
 
 const StyledDialogActions = styled(DialogActions)`
+  ${({ theme: { spacing } }) => `
   display: flex;
   justify-content: space-between;
   align-items: center;
-  gap: 24px;
+  gap: ${spacing(6)};
+`}
 `;
 
 const StyledDialog = styled(Dialog)`
-  // display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
+  ${({ theme: { palette, spacing } }) => `
   text-align: center;
+  & .MuiPaper-root {
```

Comment:
Is there a special reason we need to target this?

Reply:
We need to target inner Paper component in some way in order to style the Modal. We could use PaperProps inside Dialog to add style, but I find this cleaner. Do you have another approach in mind?

Reply:
mmmm but does either the ForegroundPaper or BackgroundPaper not fullfill this? or is that not customizable?

Reply:
I have now implemented some MUI Dialog props to improve styles definition